helm/prometheus-rules/templates/kaas/tenet/alerting-rules/node.management-cluster.rules.yml

```diff
@@ -114,3 +114,17 @@ spec:
         severity: notify
         team: {{ include "providerTeam" . }}
         topic: infrastructure
+    - alert: ManagementClusterControlPlaneNodeMemoryPressureTaint
+      annotations:
+        description: '{{`Control plane node {{ $labels.node }} has memory-pressure taint applied, indicating memory issues.`}}'
```

Comment:
So they need to be 2 diferent alerts if they are the same?

Reply:
No worries, I think the node metrics should eventually be merged in one file as having 2 alerts is not that useful :)